src/main/frontend/worker/pipeline.cljs

```diff
@@ -501,7 +501,9 @@
         (or from-disk? new-graph?)
         {:tx-report tx-report}
 
-        (or (::gp-exporter/new-graph? tx-meta) (::sqlite-export/imported-data? tx-meta))
+        (or (::gp-exporter/new-graph? tx-meta)
```

Comment:
@tiensonqin I saw that this was skipped to allow undo/redo to work for this new op. Currently when importing from a form, this is still skipped. Is there a way to call this without affecting undo/redo?

Reply:
`invoke-hooks-default` does a lot of unnecessary things to fix+adjust UI input which isn't needed when importing data. However I haven't found an issue with ref calculcation so I'm happy to leave this as is